pkg/kubelet/dockertools/manager.go

```diff
@@ -1889,3 +1891,12 @@ func (dm *DockerManager) doBackOff(pod *api.Pod, container *api.Container, podSt
 	dm.clearReasonCache(pod, container)
 	return false
 }
+
+// getPidMode returns the pid mode to use on the docker container based on pod.Spec.HostPID.
+func getPidMode(pod *api.Pod) string {
+	pidMode := ""
+	if pod.Spec.HostPID {
+		pidMode = "host"
```

Comment:
I did not see a constant for this in the docker client packages.  UTS mode and host network also are using `"host"` in `manager.go` so I went with that approach.